adding_species_manuscript.md

```diff
@@ -162,28 +164,113 @@ with stdpopsim.
+
+Any simulation requires *some* choice for 1-5,
+although values may come from another speicies;
+clearly, a neutral simulation does not require the last two.
+Furthermore, we require all ingredients added to stdpopsim
+to be documented, and citeable.
+Processes for quality control are discussed in more detail in (CITE first paper),
+but in brief (TODO: summarise).
+Here we discuss all these in more detail:
+
+A **genome assembly**,
+with contigs being assembled at the chromosome level or nearly so.
+By using chromosome level information to simulate genomes, stdpopsim indirectly simulates the effect of linked selection,
+as discussed above.
+*(move that discussion here?)*
+Although few species currently have truly chromosome-level assemblies,
+we hope that many more will be available in the near future,
+thanks to the advent of long-read sequencing technologies [ref]
+and initiatives such as B10K, XXXX, and G10K near XXX [Rhie et al, 2021, REF, REF],
+that plan to generate assemblies for XXX vertebrate species.
+
+It is also required that the species has a plausible and citeable **mutation rate** estimate.
+Both phylogenetic mutation rate [ref], germline *de novo* mutations,
+or mutation rate estimates based on mutation accumulation studies [ref] are welcomed.
+Lacking one of these,
+it is common to use an estimate from another (hopefully closely related) species.
+
+
+Since both mutation and recombination shape the genetic diversity of genomes,
+stdpopsim also requires a **recombination rate** estimate.
+Ideally, this should be a chromosome level recombination map,
+which allows more precise inference of the effect of selective interference.
+At minimum, a citeable single recombination rate estimate for the whole genome is needed.
+
+A generation time estimate, (in years), is also required.
+This is not in fact required for the simulations themselves (that usually work with units of generations),
+but is an important part of natural history
+that is required to translate results (particularly, of historical demographic models)
+into real time units.
+
+
+A simulation of a population requires a specification of that population,
+which is a primary determinant of the levels of diversity in a given population.
+Misspecification of a demographic model
+can generate highly unrealistic patterns of genetic variation,
+and so making published, quality-controlled demographic models easily available
+is one of the most important roles of stdpopsim.
+Ideally, each species will have at least one citeable **demographic model**
+(more than one because the appropriate demographic model to compare to a given dataset
+usually depends on the sampling location(s) of the data),
+although at minimum, stdpopsim requires a
+a plausible and citeable effective population size estimate,
+as a single-population simulation with a reasonable (effective) size
+should at least give comparable levels of overall genetic diversity.
+
+The goal of much population genetics work 
```

Comment:
"The above factors are sufficient for modeling neutral variation in populations, for example to infer past migration events or population size changes. However, the goal of much population genetics work..."